salt/edx/gitreload.sls

```diff
@@ -155,6 +184,6 @@ start_gitreload:
   service.running:
     - name: gitreload
     - enable: True
+    - reload: True
```

Comment:
I'm not sure whether the gitreload service supports `reload` as an option. I think we should be fine leaving it as a restart operation.